src/fides/api/alembic/migrations/versions/1ea164cee8bc_fideslang_2_data_migrations.py

```diff
@@ -270,10 +270,11 @@ def update_system_ingress_egress_data_categories(
         # Do a blunt find/replace
         if ingress:
             for item in ingress:
-                item["data_categories"] = [
-                    _replace_matching_data_label(category, data_label_map)
-                    for category in item.get("data_categories")
-                ]
+                if item["data_categories"]:
+                    item["data_categories"] = [
+                        _replace_matching_data_label(category, data_label_map)
+                        for category in item["data_categories"]
```

Comment:
I tried doing `for category in item.get("data_categories", [])` but I couldn't get it to work so I fell back to the direct `if item["data_categories"]:`

Reply:
I think I am fundamentally misunderstanding the `row` type that comes back from `sqlalchemy`